### ISSUE TITLE
ytmdesktop does not work on Fedora 35

### ISSUE THREAD

**Issue by tchellomello** [ytmdesktop/ytmdesktop#829]
- [x] I understand that **YTMDesktop have NO affiliation with Google or YouTube**.
- [x] I verified that there is no open issue for the same subject.

**Describe the bug**
ytmdesktop does not work on Fedora 35. It keeps spinning and then throws a segfault.  Installation method via `AppImage` 

**To Reproduce**
1. Run AppImage
```
./YouTube-Music-Desktop-App-1.13.0.AppImage                                                                                                                                23:25:32
Checking for update
(node:11567) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
Update for version 1.13.0 is not available (latest version: 1.13.0, downgrade is disallowed).
fish: Job 1, './YouTube-Music-Desktop-App-1.1…' terminated by signal SIGSEGV (Address boundary error)
```

**Expected behavior**
Should work as expected

**Traceback**
```
Sep 28 23:27:50 x1.tatu.home systemd-coredump[11795]: [🡕] Process 11567 (youtube-music-d) of user 1000 dumped core.
                                                      
                                                      Found module /tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app with build-id: a4cce967843200c3c13f253afeabe941837e59b9
                                                      Found module /tmp/.org.chromium.Chromium.ol3PvE with build-id: 98ebc70758e3713138579b8c1de13c6ae5557034
                                                      Found module /tmp/.org.chromium.Chromium.8wftL4 with build-id: 58d1c49c22f04fec1c744cb6211f0115bd9b3dd4
                                                      Found module /tmp/.mount_YouTubFoMm1Q/libffmpeg.so with build-id: 5784a955e496fff7bae3f27c6ec35b6fe0785fe1
                                                      Found module linux-vdso.so.1 with build-id: 89ed78c27f8a67824bd3d86be82134c9682b8777
                                                      Found module libudev.so.1 with build-id: 06534927bebf5d338c48c95f387c3579b89d84ac
                                                      Found module libdconfsettings.so with build-id: f07853974014f14e04029f524a32d4aea9558c1d
                                                      Found module libpk-gtk-module.so with build-id: 15bdc95913646361598d71c4ca1ed44fdfcc5ae3
                                                      Found module libogg.so.0 with build-id: 714c2b8484f85a852e216c4fc4bb2cf92872f37e
                                                      Found module libvorbis.so.0 with build-id: d3aece982a2dfdf9f412b5fc8a4dc88722bde200
                                                      Found module libltdl.so.7 with build-id: 4d1cbc08cd7060e81505ddc83d6e8a16d5f69583
                                                      Found module libtdb.so.1 with build-id: efecd4941ce305b2798f67e1f20c4ebd1b80cffd
                                                      Found module libvorbisfile.so.3 with build-id: 97bc441918c920e48e93a50256eb8d8b851bc741
                                                      Found module libcanberra.so.0 with build-id: 1258d29370c8d4741709eda7f2dea7ff922eb60b
                                                      Found module libgthread-2.0.so.0 with build-id: 89f4570b4481f82eef5a56ab90ad95ebfa2a656c
                                                      Found module libcanberra-gtk3.so.0 with build-id: 096164aee2aab3b6879a518dbe725f449a2b45e5
                                                      Found module libcanberra-gtk-module.so with build-id: da92365f021b6e3d58e804e4a945c25c62068b89
                                                      Found module libgvfscommon.so with build-id: 6045f9d5862c498db6342ad74543670e00bd03a3
                                                      Found module libgvfsdbus.so with build-id: dac79f425b7c59006d524fc80b4f7ec6375ccc1c
                                                      Found module libbrotlicommon.so.1 with build-id: 9e92a8ec2d9efe80ec86e04fba0549cd3202ebc4
                                                      Found module libgpg-error.so.0 with build-id: 42587ce26856ad15cf5630134f114599783477ed
                                                      Found module libgmp.so.10 with build-id: 14dbc5e93375ccb760dc47c5aaecc0cdfc45a06a
                                                      Found module libhogweed.so.6 with build-id: 288da1bb6ffaf6600c2f8a126bec247f2e9a4628
                                                      Found module libnettle.so.8 with build-id: 62dba2572638edf8336bd4de5017fc70eaf0a567
                                                      Found module libtasn1.so.6 with build-id: be32a4fab2585b6114471e83bf582b3df020b594
                                                      Found module libunistring.so.2 with build-id: edcae9db236efd11e61995054ddf27a89ec6dc40
                                                      Found module libidn2.so.0 with build-id: f3ad1b0687f5b426beed2d3d3d18aa301932f7a3
                                                      Found module libp11-kit.so.0 with build-id: a7a93824ebf14980eeb37fe23c9d86dd201f947f
                                                      Found module libresolv.so.2 with build-id: 22c0c0eb634a1a2e96f392587a07de92525686ff
                                                      Found module libcrypto.so.1.1 with build-id: 3ead607210606611d5f3398ee7e65198bd519de9
                                                      Found module libkeyutils.so.1 with build-id: 2560a16099ad1875f7ea2195ae25b97ea168a758
                                                      Found module libkrb5support.so.0 with build-id: 03649849c4d1813c307624a711b03fb15e7124cb
                                                      Found module libcom_err.so.2 with build-id: a1d791cd7600f5609702a895a64d9131d1cd7b8f
                                                      Found module libk5crypto.so.3 with build-id: 11238d51059ecf299699dc4d24a74e67a64aa725
                                                      Found module libkrb5.so.3 with build-id: 0582e999cc7d74e1ec2dc7b3c9f7aa4dae7342f6
                                                      Found module libdatrie.so.1 with build-id: 9ebd99e63729783155e081cc2c2377a767b9e3c4
                                                      Found module libbrotlidec.so.1 with build-id: 57cb8a53e33f31a620f3739919414c0d00549f87
                                                      Found module libbz2.so.1 with build-id: 19ce300b78962ee0b113afb0f1016727a2f3c37b
                                                      Found module libxml2.so.2 with build-id: 8f21e0bbe0106c5c5cd35859487dcc074b8f13bd
                                                      Found module libgraphite2.so.3 with build-id: 58f949a538f65a6221e442879144b368f663f7e3
                                                      Found module libgcrypt.so.20 with build-id: 711d41580c5a8649a79a8430a985dac3e25b5ba2
                                                      Found module libcap.so.2 with build-id: 0214aa9cc6a8646eb9ec27ab7dda6a2219da6500
                                                      Found module liblz4.so.1 with build-id: fd02c4542a1ce1ad6c503b958447be1c37a4afee
                                                      Found module libzstd.so.1 with build-id: c594c701d8f5ba174a5be306973fa9803d65e327
                                                      Found module liblzma.so.5 with build-id: 7fec53ce7cba9489f130c99cfd3ace82e9dde0ee
                                                      Found module libpcre2-8.so.0 with build-id: dbf58040a44903b428c4e68716aa4f45058ad650
                                                      Found module libblkid.so.1 with build-id: 96be27216d8d6d7ba3694ca503cd1b07f60fa539
                                                      Found module libgnutls.so.30 with build-id: 725a333b137bd282d588279e04f20d99a3ea24c1
                                                      Found module libavahi-client.so.3 with build-id: a74102919e50a8c29f9f6b7cb561d387a82761fa
                                                      Found module libavahi-common.so.3 with build-id: 07bf073b57a037f9bb8e64e12d460499ead55a7c
                                                      Found module libgssapi_krb5.so.2 with build-id: c79d44cc56f67983ba388e5336c779d9f050114e
                                                      Found module libwayland-server.so.0 with build-id: 2c1f4aa811bc973c2b48d3dee4f14ef5f57196ec
                                                      Found module libxcb-render.so.0 with build-id: 36edcc5980076552c68ccef2c6199808cba68f98
                                                      Found module libxcb-shm.so.0 with build-id: a31877f8ca8058df0005532a160d93a1104c5401
                                                      Found module libpixman-1.so.0 with build-id: dc521afa5e1d55834eebac7bb3c97cfe5e73f384
                                                      Found module libthai.so.0 with build-id: 1f7b46f526e1fba04f29535981483f50c782775e
                                                      Found module libwayland-egl.so.1 with build-id: ff42fe0eeca6a4c95efc7c2b024e98dadf763267
                                                      Found module libwayland-cursor.so.0 with build-id: 3df8fa12f532612fc50f17938906c434e86106ad
                                                      Found module libxkbcommon.so.0 with build-id: 534d0218e01a780844e80ce6544afc33731e716a
                                                      Found module libXinerama.so.1 with build-id: 72d169302fe5696536345d6da1605baeb6e3bc01
                                                      Found module libfreetype.so.6 with build-id: 6d98d390da279e5c22ebe472a552b3470554655b
                                                      Found module libfontconfig.so.1 with build-id: e50d477be88a082f6a2fd42a7db8278bb88466ca
                                                      Found module libharfbuzz.so.0 with build-id: 620a4b6e2914d6e1301c086def09a60e58450dc3
                                                      Found module libpangoft2-1.0.so.0 with build-id: 64e924e18347819f3e38600e27c18b29cd11efa0
                                                      Found module libfribidi.so.0 with build-id: a4e245984a5c2e6d842a71adea32ec7c05cb7e40
                                                      Found module libepoxy.so.0 with build-id: f368f1c90cf2ea823acc67473f816e72f368bc43
                                                      Found module libwayland-client.so.0 with build-id: ecba40c75b2d78de955fa69baada38ed183fd4f9
                                                      Found module libcairo-gobject.so.2 with build-id: 7a1381e606b2601e779ccd25afbc719c5bc4e922
                                                      Found module libcloudproviders.so.0 with build-id: 5b10ee7ce7470f15b8429030cf810e671a185de5
                                                      Found module libjpeg.so.62 with build-id: f50d5361714c8fe58ed898123a98aa0d6547068e
                                                      Found module libpng16.so.16 with build-id: afcd45d29b0d579615db44a29213544328adde6f
                                                      Found module libsystemd.so.0 with build-id: 9b9450f68ebd1030091d1dc0f15a51e4cb1360fc
                                                      Found module libplds4.so with build-id: 3aa374aa1cb69f836123ed13b8aabfeddd94fcab
                                                      Found module libplc4.so with build-id: cb2466b5028118d834c86746815d966e56fe2615
                                                      Found module libXau.so.6 with build-id: b6a2d4859848c28bf0a4fc8744d04b1935879af6
                                                      Found module libselinux.so.1 with build-id: 104040a9146de3345eebd544f253ad4f9f6cc8c7
                                                      Found module libmount.so.1 with build-id: 7f7bccf74cb835b64429e61ea0ec03cd0cb95b8b
                                                      Found module libz.so.1 with build-id: 5903f5c355c264403e4e7cdc66779584425ca3b8
                                                      Found module libgmodule-2.0.so.0 with build-id: 194e4d8da706fd9452a6fff36459b7ffbeeb5091
                                                      Found module libpcre.so.1 with build-id: fcbdb4731e50f20a47b62341a6841dd616fe495d
                                                      Found module libffi.so.6 with build-id: 0129f72b58e11d59546f7f207d2c90af7ebd9a5e
                                                      Found module ld-linux-x86-64.so.2 with build-id: 7d2c06364366496318a96401aee1c204f80ee73a
                                                      Found module libc.so.6 with build-id: 091494afec47a9dec3af480bf1ccfd55d5376e0c
                                                      Found module libgcc_s.so.1 with build-id: 279c1648ee67377b5de1c45a244740beea48423e
                                                      Found module libcups.so.2 with build-id: 6c3c72cc2dde5487dab43cb21e34543bfdfeb16f
                                                      Found module libatspi.so.0 with build-id: 93de58828b176bf6eb76897e1cb2e576a1d5fddf
                                                      Found module libasound.so.2 with build-id: a5921ca91384889796cbc7eb4b4ba21964222320
                                                      Found module libgbm.so.1 with build-id: 9300f85fb87b609844147ead7717208af8a417b5
                                                      Found module libdrm.so.2 with build-id: fa61ece3513476ba39a7b81d409d623748c03dc0
                                                      Found module libexpat.so.1 with build-id: de380d46dd01700b75b3cecd611e3ec7da062253
                                                      Found module libXrandr.so.2 with build-id: eb8b304d315d3ee09980cafe7ceda9582daa40e8
                                                      Found module libm.so.6 with build-id: fed79663436256eb88044bf3cff419c08487da77
                                                      Found module libcairo.so.2 with build-id: 8a28f43397104b4ef517df6c119772b5d577f48e
                                                      Found module libpango-1.0.so.0 with build-id: 22475165c324d18f5b1deb22fd8c6808b5e605ea
                                                      Found module libpangocairo-1.0.so.0 with build-id: 78f494b4f660cc741ed08937ddd209adc0c85e34
                                                      Found module libgdk-3.so.0 with build-id: cce7a3b33974de227f0373beb0addec993964069
                                                      Found module libgtk-3.so.0 with build-id: e3452fbe01458a7dc1eaab3b020174fd2911dea7
                                                      Found module libgdk_pixbuf-2.0.so.0 with build-id: b6e5e7c59ce90aecabf6b0b005ecd138dd6bae45
                                                      Found module libdbus-1.so.3 with build-id: 450ef390503ec82a65caa90c841d4068d136dcf3
                                                      Found module libatk-bridge-2.0.so.0 with build-id: 8df66b2fcb2033cdad8e26a3ad13ad7f3cdf4144
                                                      Found module libatk-1.0.so.0 with build-id: b3d190657efe01bedd3077177d2bd473b7b737df
                                                      Found module libnspr4.so with build-id: ef2df98a4b055d18ecc8d6220ca3625fce83e9a1
                                                      Found module libsmime3.so with build-id: f0976a90d66c298ff85ac25e7c8b6c1237d15b68
                                                      Found module libnssutil3.so with build-id: 1b3f8b33e4d967829322c7d2dacf4fe50ad58a7e
                                                      Found module libnss3.so with build-id: 0549f5ac040f8cc7e7ae42ad12e3d24ea5d448f2
                                                      Found module libXrender.so.1 with build-id: 45e135fd2ee2393b7eb0c94f964542b8326582e1
                                                      Found module libXi.so.6 with build-id: e1d4fcbcab03948da50a0e175ac2459c335e7939
                                                      Found module libXfixes.so.3 with build-id: e8020da14577d275b75ed0c4cb18b1050fc33595
                                                      Found module libXext.so.6 with build-id: 08e82aa07cb7910a08d8386fb16de7896ce290f4
                                                      Found module libXdamage.so.1 with build-id: 7ad2a44d600f984d1728b624701f5d41bba04067
                                                      Found module libXcursor.so.1 with build-id: fb3d6eb4c582d4761eab821bc0ceb77fc7c2654f
                                                      Found module libXcomposite.so.1 with build-id: 4223e166fff31d2e529cc5f856c7100fbdf67cd9
                                                      Found module libxcb-dri3.so.0 with build-id: 0f4fd31b851a10e0ac9726faf3b2b66a52a56162
                                                      Found module libxcb.so.1 with build-id: 1352e33391b3079347f320218799d6e531bd42f1
                                                      Found module libX11-xcb.so.1 with build-id: c9ad3de1db0ce5fadbbba3d0dfa3346777547678
                                                      Found module libX11.so.6 with build-id: 8d35504e71458ad6afe46c6e64b45f56e752e904
                                                      Found module libgio-2.0.so.0 with build-id: 871adc0d488cf2e1b59821604b134623c5e825f4
                                                      Found module libglib-2.0.so.0 with build-id: d8a7fdad310ada7c7d2ea0820ee5e6b555a9f1ce
                                                      Found module libgobject-2.0.so.0 with build-id: bbd30c8f7fde96b24695d46d412293c2178bddac
                                                      Found module librt.so.1 with build-id: a9008e4cbe33ce59649f0b5df29094b648d140e5
                                                      Found module libpthread.so.0 with build-id: 9dd6d13a591d76aec1636e893d579fdc5732996f
                                                      Found module libdl.so.2 with build-id: dba6ce503193c7be2c60939b2d893983e7a669f4
                                                      Stack trace of thread 11567:
                                                      #0  0x000055642848f8bf n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x19618bf)
                                                      #1  0x0000556428424c36 n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x18f6c36)
                                                      #2  0x0000556428424b2c n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x18f6b2c)
                                                      #3  0x000055642848a898 n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x195c898)
                                                      #4  0x000055642d643c89 n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x6b15c89)
                                                      #5  0x000055642d6664a7 n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x6b384a7)
                                                      #6  0x000055642d6846bf n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x6b566bf)
                                                      #7  0x000055642d680c54 n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x6b52c54)
                                                      #8  0x000055642d682868 n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x6b54868)
                                                      #9  0x000055642d641e0b n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x6b13e0b)
                                                      #10 0x00005564284ccc2e n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x199ec2e)
                                                      #11 0x00005564284683b6 n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x193a3b6)
                                                      #12 0x00005564283df66b n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x18b166b)
                                                      #13 0x00005564291509b7 n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x26229b7)
                                                      #14 0x0000556429b089f8 n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x2fda9f8)
                                                      #15 0x0000556429a9dcb5 n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x2f6fcb5)
                                                      #16 0x0000556429a9dcb5 n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x2f6fcb5)
                                                      #17 0x0000556429a9509f n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x2f6709f)
                                                      #18 0x0000556429a9b7fa n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x2f6d7fa)
                                                      #19 0x0000556429a9b5d8 n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x2f6d5d8)
                                                      #20 0x00005564292006fd n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x26d26fd)
                                                      #21 0x0000556429200426 n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x26d2426)
                                                      #22 0x00005564290fe5d0 n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x25d05d0)
                                                      #23 0x000055642e0a11fb n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x75731fb)
                                                      #24 0x000055642e0a1451 n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x7573451)
                                                      #25 0x00005564284f63cd n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x19c83cd)
                                                      #26 0x00005564283ebdde n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x18bddde)
                                                      #27 0x00005564283ea190 n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x18bc190)
                                                      #28 0x000055642842492e n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x18f692e)
                                                      #29 0x000055642848b84b n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x195d84b)
                                                      #30 0x0000556428490cc0 n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x1962cc0)
                                                      #31 0x000055642d6419b1 n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x6b139b1)
                                                      #32 0x000055642d6771dd n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x6b491dd)
                                                      #33 0x000055642d6709eb n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x6b429eb)
                                                      #34 0x000055642b602652 n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x4ad4652)
                                                      #35 0x000055642b602a63 n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x4ad4a63)
                                                      #36 0x000055642b602c50 n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x4ad4c50)
                                                      #37 0x000055642b4d948b n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x49ab48b)
                                                      #38 0x000055642b609bcc n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x4adbbcc)
                                                      #39 0x00007fc04116e33f g_main_context_dispatch (libglib-2.0.so.0 + 0x5533f)
                                                      #40 0x00007fc0411c3288 g_main_context_iterate.constprop.0 (libglib-2.0.so.0 + 0xaa288)
                                                      #41 0x00007fc04116b9e3 g_main_context_iteration (libglib-2.0.so.0 + 0x529e3)
                                                      #42 0x000055642a9a2422 n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x3e74422)
                                                      #43 0x000055642a9e28a4 n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x3eb48a4)
                                                      #44 0x000055642a9bb9be n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x3e8d9be)
                                                      #45 0x0000556429e4436a n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x331636a)
                                                      #46 0x0000556429e46272 n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x3318272)
                                                      #47 0x0000556429e4153d n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x331353d)
                                                      #48 0x0000556429d1e7a0 n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x31f07a0)
                                                      #49 0x0000556429d1e21b n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x31f021b)
                                                      #50 0x000055642be9259f n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x536459f)
                                                      #51 0x000055642901b751 n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x24ed751)
                                                      #52 0x00005564283cc4bd n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x189e4bd)
                                                      #53 0x00007fc03f8bb560 __libc_start_call_main (libc.so.6 + 0x2d560)
                                                      #54 0x00007fc03f8bb60c __libc_start_main@@GLIBC_2.34 (libc.so.6 + 0x2d60c)
                                                      #55 0x00005564283cc1ea n/a (/tmp/.mount_YouTubFoMm1Q/youtube-music-desktop-app + 0x189e1ea)
```

**Environment (please complete the following information):**
 * YTMDesktop version: ***1.13.0***
 * OS: ***Linux*** 
 * OS version: ***Fedora 35***
 * Arch: ***x64***
* Installation way: ***AppImage***

**Comment by tchellomello** [comment]
```bash
[ 1031.039298] youtube-music-d[11567]: segfault at fffffffd54debd11 ip 000055642848f8bf sp 00007ffdc32ba8f0 error 5 in youtube-music-desktop-app[5564283cc000+5f85000]
[ 1031.039340] Code: 57 41 56 53 50 48 85 f6 74 45 48 89 fb 4c 8b bf e8 00 00 00 4d 85 ff 74 36 49 89 f6 48 89 f7 e8 17 a8 ff ff 48 8b 08 48 89 c7 <ff> 51 10 4c 89 ff 48 89 c6 e8 93 ca 19 05 4c 89 75 e0 48 81 c3 38
```

**Comment by tchellomello** [comment]
Starting the app with `--no-sandbox` loaded. Using as a workaround for now
```
./YouTube-Music-Desktop-App-1.13.0.AppImage --no-sandbox              23:55:53
Checking for update
(node:14428) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
Update for version 1.13.0 is not available (latest version: 1.13.0, downgrade is disallowed).
(node:14428) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 change listeners added to [EventEmitter]. Use emitter.setMaxListeners() to increase limit

```

**Comment by ProjectSynchro** [comment]
This looks to have been fixed upstream: https://github.com/electron/electron/pull/31091 and is released in electron `v13.5.0`

**Comment by Alipoodle** [comment]
Closing issue due to no longer supportinng YouTube Music Desktop v1 + Release of YouTube Music v2.
Please re-open this issue if this is still occuring in the latest build of the application, however as a re-write has been done on the entire project most issues will be fixed, no longer a feature or re-worked.
